shared/Microsoft.AspNetCore.Routing.UrlMatchingTree.Sources/UrlMatchingNode.cs

```diff
@@ -51,31 +81,26 @@ public UrlMatchingNode(int length)
 
         /// <summary>
         /// Gets or sets the <see cref="UrlMatchingNode"/> representing
-        /// parameter segments with constraints following this segment in the <see cref="TreeRouter"/>.
+        /// parameter segments with constraints following this segment.
```

Comment:
I thought about removing comments but in Routing this API is public.

Reply:
That's fine